go-controller/pkg/ovn/egressfirewall_dns.go

```diff
@@ -84,7 +84,7 @@ func (e *EgressDNS) Add(namespace, dnsName string) (addressset.AddressSet, error
 
 }
 
-func (e *EgressDNS) Delete(namespace string) bool {
+func (e *EgressDNS) Delete(namespace string) (bool, error) {
```

Comment:
Do we still need to return a boolean? I don't see it being used anywhere :)

Reply:
not really it wasn't even used b4 my changes I will just keep err